Release-Process.md

```diff
@@ -86,6 +86,14 @@
     ```
     The above step shall create OBS packages for Kata for various distributions that Kata supports and test them as well.
   - Verify that the packages have built successfully by checking the [Kata OBS  project page](https://build.opensuse.org/project/subprojects/home:katacontainers).
+  - Make sure packages are working manually or via [package testing pipeline](http://jenkins.katacontainers.io/job/package-release-testing).
```

Comment:
suggested rephrase: `Make sure packages work correctly. This can be done manually or via the package testing pipeline.`